code/game/dna/dna2.dm

```diff
@@ -143,8 +143,6 @@ var/global/list/facial_hair_styles_female_list	= list()
 		new_dna.SE[b]=SE[b]
 		if(b<=DNA_UI_LENGTH)
 			new_dna.UI[b]=UI[b]
-	new_dna.UpdateUI()
-	new_dna.UpdateSE()
```

Comment:
UpdateUI and UpdateSE loop over the SE and UI lists to generate strings out of the hex representation of those values. Since we're cloning the datum, it's not necessary to do that work, we just copy them from the existing datum.
Writing this comment, I realized `struc_enzymes` is not being copied while `unique_enzymes` is. I'll fix this now.